### ISSUE TITLE
1.0.5

### ISSUE THREAD

**Issue by janwirth** [FranzSkuffka/vue-medium-editor#31]
There are a few things scheduled for release #1.0.5
- [x] native event MUST be used `subscribe('editableContent')` AND MUST be detached on component destruction.
- [x] babili hint MUST be added to documentation
- [x] NPM package must be fixed with regard to unexpected token import issue. See #30
- [x] babili MUST be used in the build in addition to uglify
- [x] medium editor object MUST be passed with every emitted event so that every user can do their own fancy stuff during events
- [x] min script inside build script MUST be fixed
- [ ] note about https://github.com/yabwe/medium-editor/issues/1153 must be added to readme.
- [ ] Docs, & Playgroud / demo site must be updated.
- [ ] automated version bump & build SHOULD be combined. See #33
- [ ] automated testing SHOULD be analysed with regard to feasibility AND implemented if possible
- [ ] MAJOR version MUST be bumped

**Comment by janwirth** [comment]
Currently we are blocked by some VDom issue - or a misuse as implemented in vue-medium-editor. See the progress here: https://github.com/vuejs/vue/issues/5519

**Comment by janwirth** [comment]
Discovered a new issue during development of the latest version:
https://github.com/yabwe/medium-editor/issues/1153
